src/serverenvironment.cpp

```diff
@@ -417,6 +417,11 @@ void ServerEnvironment::init()
 	std::string auth_backend_name = "sqlite3";
 
 	bool succeeded = conf.readConfigFile(conf_path.c_str());
+	std::string blocksize = conf.exists("blocksize") ?
+		conf.get("blocksize") : "16";
+	if (blocksize != std::to_string(MAP_BLOCKSIZE)) {
```

Comment:
```suggestion
	u16 blocksize = 16;
	if (conf.getU16NoEx("blocksize", blocksize) && blocksize != MAP_BLOCKSIZE) {
```
String comparisons on values that are supposed to be numeric are evil (error-prone).

Reply:
Fair enough. I was internally debating that, then - misguidedly - wanted to avoid an int parse :)
I'll change it.

Reply:
Done. Also moved it into the succeeded if statement.